exchange/utils.go

```diff
@@ -761,10 +761,11 @@ func setUserExtWithCopy(request *openrtb2.BidRequest, userExtJSON json.RawMessag
 
 // resolveBidder returns the known BidderName associated with bidder, if bidder is an alias. If it's not an alias, the bidder is returned.
 func resolveBidder(bidder string, aliases map[string]string) openrtb_ext.BidderName {
+	normalisedBidderName, _ := openrtb_ext.NormalizeBidderName(bidder)
```

Comment:
should check if `normalisedBidderName`  is returned from `openrtb_ext.NormalizeBidderName`

Reply:
At this point in code - since we have already validated imp ext using `deps.validateImpExt`, there is no need to re-check the normalisedBidderName. Moreover, this could be an alias in the request as returned in line 766.

Reply:
> At this point in code - since we have already validated imp ext using `deps.validateImpExt`, there is no need to re-check the normalisedBidderName. Moreover, this could be an alias in the request as returned in line 766.

Would like to know your thoughts from  `resolveBidder` as standalone function. If this function is called from other caller  without using using `deps.validateImpExt`. Then is it be safe to skip check on line 764

Reply:
also could we reuse `resolveBidder` in other places where look up is being done

Reply:
As of today, `resolveBidder` is package safe so it is not exposed to rest of the packages. This function is also not being used in any other places than the auction flow. In auction flow, we are already making the check using `deps.validateImpExt` function. 
Also test cases are being written with the assumption that resolveBidder returns the bidderName since there is no failure that can happen in resolveBidder function. Changing the function signature will add lot of those changes too.

From the standalone function perspective, there could be multiple cases that should be included to make this function safe to be used as standalone for example - to check if the bidder is disabled.